src/mprismanager.h

```diff
@@ -26,30 +23,31 @@
 #ifndef MPRISMANAGER_H
 #define MPRISMANAGER_H
 
-#include <mprisqt.h>
 #include <Mpris>
 #include <MprisController>
 
 #include <QDBusConnection>
 #include <QDBusObjectPath>
 
-#include <QtCore/QByteArray>
-#include <QtCore/QList>
-#include <QtCore/QMap>
-#include <QtCore/QObject>
-#include <QtCore/QSharedPointer>
-#include <QtCore/QString>
-#include <QtCore/QStringList>
-#include <QtCore/QVariant>
+#include <QByteArray>
+#include <QList>
+#include <QObject>
+#include <QString>
+#include <QStringList>
+
+namespace Amber {
+
+class MprisMetaData;
+class MprisManagerPrivate;
 
-class QSignalMapper;
 class MPRIS_QT_EXPORT MprisManager : public QObject
 {
     Q_OBJECT
 
     Q_PROPERTY(bool singleService READ singleService WRITE setSingleService NOTIFY singleServiceChanged)
```

Comment:
Existing thing, but this seems an interesting property and not entirely sure how to interpret it. The app developer set separately set currentService and singleService but what do true or false on the latter mean...